### ISSUE TITLE
Files without runs?

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/climate-explorer-backend#109]
In adding tests for the `streamflow/watershed` API, I discovered that if a file does not have a run associated with it, then the `models` API endpoint fails. The key code is this:

```
{dfv.file.run.model.short_name for dfv in ensemble.data_file_variables}
```

where `ensemble` is a `pycds.Ensemble` object.

Overlooking the fact that there are likely alternative formulations more robust to absent links between ensemble and model, there's the question of whether we already do or plan to allow for files without runs. I am not sure how we resolved this question, so this issue may just be a placeholder for retrieving a previous decision that resolves this problem while I continue working on the watershed API.

**Comment by corviday** [comment]
I think in the past we've specified the "run" for files that are not model output as either "na" or "base", but we've given them runs.

**Comment by rod-glover** [comment]
Thank you, @corviday , working when you are supposed to be resting!

**Comment by jameshiebert** [comment]
Yeah, I'd argue that we should only be querying files that are linked to the `runs` table (and that non-model output should at least have *something*). If you both agree, then this could make a [good-first-issue](https://github.com/pacificclimate/climate-explorer-backend/issues?q=is%3Aissue+is%3Aopen+label%3Agood-first-issue).
